var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -12,8 +15,8 @@ class Crtm(CMakePackage):
     scattering, and a solver for a radiative transfer."""
 
     homepage = "https://www.jcsda.org/jcsda-project-community-radiative-transfer-model"
-    git = "https://github.com/JCSDA/crtm.git"
-    url = "https://github.com/JCSDA/crtm/archive/refs/tags/v2.3.0.tar.gz"
+    git = "https://github.com/JCSDA/crtmv3.git"
```

Comment:
Restore the old repository; otherwise, all of the commits for `2.3.0` through `v2.4-jedi.2` cannot be confirmed.